src/Test/L1/Worker/L1TestBase.cs

```diff
@@ -0,0 +1,617 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.TeamFoundation.DistributedTask.Pipelines;
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
+using Microsoft.VisualStudio.Services.Agent.Util;
+using Microsoft.VisualStudio.Services.Agent.Worker;
+using Microsoft.VisualStudio.Services.Agent.Worker.Build;
+using Microsoft.VisualStudio.Services.Agent.Worker.Release;
+using Microsoft.VisualStudio.Services.Agent.Worker.Telemetry;
+using Microsoft.VisualStudio.Services.WebApi;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+using Pipelines = Microsoft.TeamFoundation.DistributedTask.Pipelines;
+
+namespace Microsoft.VisualStudio.Services.Agent.Tests.L1.Worker
+{
+    public class TestResults
+    {
+        public int ReturnCode { get; internal set; }
+        public TaskResult Result { get; internal set; }
+        public bool TimedOut { get; internal set; }
+    }
+
+    public class L1TestBase
+    {
+        protected TimeSpan _channelTimeout = TimeSpan.FromSeconds(100);
```

Comment:
Nit: is this the right naming convention?

Reply:
Fixed